src/main/kotlin/com/anaplan/engineering/vdmgradleplugin/TypeChecker.kt

```diff
@@ -50,25 +51,41 @@ open class VdmTypeCheckTask(private val includeTests: Boolean) : OvertureTask()
         @InputFiles
         get() = project.locateAllSpecifications(dialect, includeTests)
 
-    val generatedLibFile: File
+    // Only needed to ensure caching of the Gradle task
+    val statusFile: File
         @OutputFile
-        get() = project.generatedLibFile
-
+        get() = File(project.vdmBuildDir, "typeCheckStatus.log")
 
     override fun exec() {
         logger.info("VDM dialect: $dialect")
-        jvmArgs = project.vdmConfig.overtureJvmArgs
-        super.setArgs(constructArgs())
-        super.setClasspath(project.files(createClassPathJar()))
-        super.exec()
+        if (specificationFiles.isEmpty) {
+            logger.info("No files found")
+        } else {
+            jvmArgs = project.vdmConfig.overtureJvmArgs
```

Comment:
here
`            val output = ByteArrayOutputStream();
            standardOutput = output
`
L68
`
            File("f").writeBytes(output.toByteArray())
`
(you could do away with `output` if you're happy to cast)

Reply:
This completely redirects the output, though. I would have liked a `tee` equivalent, i.e. a way to both print the output to stdout and save it to a log file.